api/src/main/java/io/strimzi/api/kafka/model/EntityUserOperatorSpec.java

```diff
@@ -39,6 +39,8 @@
     public static final int DEFAULT_ZOOKEEPER_PORT = 2181;
     public static final long DEFAULT_FULL_RECONCILIATION_INTERVAL_SECONDS = 120;
     public static final long DEFAULT_ZOOKEEPER_SESSION_TIMEOUT_SECONDS = 6;
+    public static final int DEFAULT_CERTS_VALIDITY_DAYS = 365;
+    public static final int DEFAULT_CERTS_RENEWAL_DAYS = 30;
```

Comment:
We are starting to have these two values all around the operators / api.
The `ModelUtils` class for example is using the constant from the `AbstractModel` about the cert validity days while it has an hard coded "30" value for the cert renewal days.
Are the above constants in the `api` module because we want them to be available through the api? In that case, because the validity is the same is the `EntityUserOperatorSpec` the right place?